pkg/services/shorturls/short_url_service.go

```diff
@@ -42,6 +44,18 @@ func (s ShortURLService) GetShortURLByUID(ctx context.Context, user *models.Sign
 	return &shortURL, nil
 }
 
+func (s ShortURLService) UpdateLastSeenAt(ctx context.Context, shortURL *models.ShortUrl) error {
+	shortURL.LastSeenAt = getTime().Unix()
+	return s.SQLStore.WithTransactionalDbSession(ctx, func(dbSession *sqlstore.DBSession) error {
+		_, err := dbSession.Id(shortURL.Id).Update(shortURL)
```

Comment:
```suggestion
		_, err := dbSession.ID(shortURL.Id).Update(shortURL)
```
for fixing the deprecation warning.